src/constants.js

```diff
@@ -5,7 +5,8 @@ export const INITIAL_SESSION_TIMEOUT = 5
 export const INITIAL_NONCE = 0;
 export const TRANSFER_ATTACHMENT_BYTE_LIMIT = 140;
 export const TX_TEST_EXPLORER = 'https://testexplorer.v.systems/transactions/'
```

Comment:
TX_TEST_EXPLORER need to remove. All transactions checking is using `EXPLORER  + TX_URL` now.